kolibri/content/management/commands/importchannel.py

```diff
@@ -65,12 +65,13 @@ def _transfer(self, method, channel_id, path=None):
 
         with filetransfer:
 
-            with self.start_progress(total=filetransfer.total_size) as progress_update:
+            with self.start_progress(total=filetransfer.total_size*1.1) as progress_update:
 
                 for chunk in filetransfer:
                     progress_update(len(chunk), progress_extra_data)
 
-        annotation.update_channel_metadata_cache()
+                channel_import.import_channel_from_local_db(channel_id)
+                progress_update(filetransfer.total_size*0.1, progress_extra_data)
```

Comment:
I don't have strong opinions on either, but it would make more sense for me if importing the DB (`importchannel`) took an amount of the progress bar proportional to the amount of bytes needed to import the entire channel.

For downloading from the internet, that's hard to determine upfront. Since downloading actual content is the majority of the transfer time, we simply showed the total progress for *just* importing content, rather than trying to include the content DB in it. 

I would prefer to just omit progress data for the content DB, since it seems to be misinforming progress (in case they're downloading multi-GB channels).

Reply:
OK, my only concern was that it could take a bit of time, and it is misleading to the user to show full completion when it is not. Can remove it for now, and we can rethink the UX at a later time.